src/c4/Global_Timer.cc

```diff
@@ -130,7 +130,7 @@ void Global_Timer::report_all(ostream &out) {
         }
       }
     }
-    for (unsigned i = 0; i < 80; ++i)
+    for (unsigned i = 0; i < 92; ++i)
       cout << '-';
```

Comment:
New code was added to this PR, so I will review the new portions:
* Maybe use `cout << string(92,'-'); ` instead of the 2 loops above.